tests/conftest.py

```diff
@@ -46,51 +60,102 @@ def docker_alluxio():
         "-Dalluxio.master.journal.type=NOOP "
         "-Dalluxio.master.scheduler.initial.wait.time=1s "
         "-Dalluxio.dora.client.ufs.root=file:/// "
-        '-Dalluxio.underfs.xattr.change.enabled=false " alluxio/alluxio:308-SNAPSHOT master'
+        + (
+            "-Dalluxio.worker.membership.manager.type=ETCD "
+            "-Dalluxio.etcd.endpoints=http://etcd:2379 "
+            if is_etcd
+            else ""
+        )
+        + '-Dalluxio.underfs.xattr.change.enabled=false " alluxio/alluxio:310-SNAPSHOT master'
     )
+
     run_cmd_worker = (
-        "docker run --platform linux/amd64 -d --rm --net=alluxio_network -p 28080:28080 -p 29999:29999 -p 29997:29997 "
+        "docker run --platform linux/amd64 -d --rm -h localhost --net=alluxio_network -p 28080:28080 -p 29999:29999 -p 29997:29997 "
```

Comment:
why -h localhost is needed for worker but not for master?

Reply:
why need localhost parameter?

Reply:
So when worker register itself into etcd? It uses its own hostname to register. That hostname will be its container hash without specifying the hostname here. So the python client will not get the correct worker host from etcd.

Reply:
@ssz1997 is it the recommended way?

Reply:
@LuQQiu Tried another way, use alluxio config to do that.